src/GUI/GUI.java

```diff
@@ -450,13 +450,16 @@ private static void addSaveCalculationButton() {
             public void actionPerformed(ActionEvent e) {
                 SwingUtilities.invokeLater(new Runnable() {
                     public void run() {
-                        LastCalculation.setConfigLastCalc(baseCurResult, targetCurResult, inputValue);
-                        runFadeLabel();
+                        if (inputValue == null || baseCurResult == null || targetCurResult == null) {
+                            PopupDisplay.throwErrorPopup("Es gibt nichts zum Speichern!", "");
```

Comment:
PopupDisplay muss hier später umgeändert werden, nachdem #59 in main ist

Reply:
#59 ist approved